includes/Admin/Orders.php

```diff
@@ -323,15 +323,18 @@ private function get_cancel_modal_buttons() {
 	 * @internal
 	 *
 	 * @since 2.1.0-dev.1
+	 *
+	 * @param boolean $select_id id for the select HTML element
+	 * @param boolean $hidden whether or not the field should be hidden
 	 */
-	public function render_refund_reason_field() {
+	public function render_refund_reason_field( $select_id = 'wc_facebook_refund_reason', $hidden = true ) {
 
 		if ( ! $this->is_edit_order_screen() ) {
 			return;
 		}
 
 		?>
-		<select id="wc_facebook_refund_reason" style="display: none;">
+		<select id="<?php echo $select_id; ?>" <?php echo $hidden ? 'style="display: none;"' : ''; ?>>
```

Comment:
```suggestion
		<select id="<?php echo esc_attr( $select_id ); ?>" <?php echo $hidden ? 'style="display: none;"' : ''; ?>>
```
Not a big deal, but for the sake of safety.